config/routes.rb

```diff
@@ -2,4 +2,5 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :posts
+  resources :likes
```

Comment:
Can you nest the resources instead:

```ruby
resources :posts do
  resources :likes
end
```

It may even be a good idea to nest the user as well..

```ruby
resources :users do
  resources :posts do
    resources :likes
  end
end
```